### ISSUE TITLE
Empty Columns on Dashboard

### ISSUE THREAD

**Issue by edru** [Hellowlol/HTPC-Manager#27]
Modules Loaded: Show recent from Kodi: movies, Shows, Music, CouchPotato, Sabnzbd, Sickrage
![capture](https://cloud.githubusercontent.com/assets/1192974/6132661/ab793fae-b119-11e4-82cf-62b2e20a6bc0.PNG)

Ignore the empty Kodi spots as I need to configure Kodi. Just wondering why Next Aired wouldnt be in column 3 of the 2nd row as it appears empty.

**Comment by Hellowlol** [comment]
Ah, it doesn't work as your db referes to xbmc while the code uses kodi. Please add a new server

**Comment by edru** [comment]
This is all set up from scratch with a brand new install. So all this is
brand new. Is there a config file hiding somewhere else I should have
deleted?

**Comment by Hellowlol** [comment]
Nope, but a assumed you copied your old db?

**Comment by edru** [comment]
Nah. I just fully deleted the entire HTPC Manger folder from the previous
repo. I then synced yours to a brand new folder. The only thing that didnt
get changed from the old to the new is the init script. Figured that didnt
change.

**Comment by edru** [comment]
To clarify, im not concerned about why the kodi stuff is empty, I know why
its empty. Im more wondering why there is an empty spot on the 2nd row on
the right.

**Comment by Hellowlol** [comment]
That nothing to worry about, it's suppose to be like that if you haven't activated that module.

**Comment by edru** [comment]
A little confused then. If i disable something else, the columns move

![capture1](https://cloud.githubusercontent.com/assets/1192974/6133571/78fbc7e8-b120-11e4-8181-02701ba49227.PNG)

Another example of columns moving.

![capture2](https://cloud.githubusercontent.com/assets/1192974/6133616/b3a88390-b120-11e4-8481-7a99968cc3ad.PNG)

**Comment by Hellowlol** [comment]
tl;dr Its suppose to be like this. Like i have said before, the dash should be rewritten

Longer version:

There are no columns, but rows.

In each row there are 3 modules, each of this float left and are the 1/3 of the rows width

So say you have a row with:
Latest Movies, Latest Shows, Latest Music



If you disable Latest Movies the row will then be. 
Latest Shows, Latest Music, BLANK

Styxit version version solves (bug) does this by overloading the row with more then 3 modules, because it isnt enough space the the margin and gutter get all fucked up(looks ulgy). I hope this helps.

**Comment by edru** [comment]
Ahh got yah. Thanks man.
